aiohttp/client.py

```diff
@@ -1269,6 +1273,22 @@ async def __aexit__(
         await self.close()
 
 
+def get_auth_from_netrc(url):
+    """Get authentication through netrc."""
+    auth = None
+    netrc_obj = netrc_from_env()
+    if netrc_obj:
+        auth_from_netrc = netrc_obj.authenticators(url.host)
+        if auth_from_netrc is not None:
+            # auth_from_netrc is a (`user`, `account`, `password`) tuple,
+            # `user` and `account` both can be username,
+            # if `user` is None, use `account`
+            *logins, password = auth_from_netrc
```

Comment:
Why use a tuple if it's a fixed size? I think this is easier to read using 3 variables (at which point the first comment line in unneeded, as the code would already describe this).

Reply:
Fixed in [`5f4d4dc` (#9133)](https://github.com/aio-libs/aiohttp/pull/9133/commits/5f4d4dc0b0c7071b2d3a30398706d862d7e4536c)